Adapt expose tutorial and add multi namespace one (#15574)
kyma-project/kyma: docs/03-tutorials/00-api-exposure/apix-03-expose-workload-apigateway.md

   >**NOTE:** If you are running Kyma on k3d, add `httpbin.kyma.local` to the entry with k3d IP in your system's `/etc/hosts` file.

>    >**NOTE:** You may specify a Namespace for your service. If you omit it, the default is then the APIRule Namespace.

Reviewer: ```suggestion
   >**NOTE:** If you didn't specify a Namespace for your service, the default APIRule Namespace will be used.
```
